tests/foreman/ui/test_host.py

```diff
@@ -2343,3 +2343,27 @@ def test_positive_cockpit(session, default_sat):
         ), 'cockpit page shows hostname {} instead of {}'.format(
             hostname_inside_cockpit, default_sat.hostname
         )
+
+
+@pytest.mark.tier4
+def test_positive_read_details_page_from_new_ui(session, module_host_template):
```

Comment:
Case importance should be critical for this test?

Reply:
why isn't the high, high enough?

Reply:
I would leave importance designations up to the component owners.

Reply:
IMHO, reading details from the host details(that too its a new UI) page should be treated as necessary functionality and hence it should be critical.

I will also leave it up to @tstrych to decide, it was just my opinion.

Reply:
now it's critical, year after it is low, but all tests importances are something that is not changing much even though we as component owners can do that, but it is rock bottom priority for me.